evap/static/ts/src/datagrid.ts

```diff
@@ -158,7 +158,12 @@ abstract class DataGrid {
             header.classList.remove("col-order-asc", "col-order-desc");
         }
         for (const [column, ordering] of this.state.order) {
-            this.sortableHeaders.get(column)!.classList.add(`col-order-${ordering}`);
+            const header = this.sortableHeaders.get(column);
+            if (header === undefined) {
+                // Silently ignore non-existing columns: They were probably renamed, users will have to rebuild their localstorage entry
```

Comment:
And this will happen when? Can we rephrase the comment to rather emphasize why this is not a problem?

Reply:
Rephrased to make clear that this will fix itself "automatically" the next time the user sorts by a column.